plugins/greynoise/icon_greynoise/actions/riot_lookup/action.py

```diff
@@ -17,10 +17,17 @@ def run(self, params={}):
             resp = self.connection.gn_client.riot(params.get(Input.IP_ADDRESS))
             if resp["riot"]:
                 resp.pop("logo_url", None)
-                resp["viz_url"] = "https://viz.greynoise.io/riot/" + str(params.get(Input.IP_ADDRESS))
+                resp["viz_url"] = "https://viz.greynoise.io/ip/" + str(params.get(Input.IP_ADDRESS))
```

Comment:
Can you make `"https://viz.greynoise.io/ip/"` a defined variable above, then call the variable here instead?